omise/classes/payment_order.php

```diff
@@ -0,0 +1,117 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+class PaymentOrder extends ModuleFrontController
```

Comment:
@nimid If classes/payment_order.php is a controller, I think it better to keep it in controllers folder. (saw that this class extends a parent class from `ModuleFrontController`).